qemu_v8.mk

```diff
@@ -261,6 +286,9 @@ linux-cleaner: linux-cleaner-common
 # OP-TEE
 ################################################################################
 OPTEE_OS_COMMON_FLAGS += DEBUG=$(DEBUG) CFG_ARM_GICV3=$(GICV3)
+ifeq ($(XEN_BOOT),y)
+OPTEE_OS_COMMON_FLAGS += CFG_VIRTUALIZATION=y CFG_CORE_ASLR=y CFG_TA_ASLR=y CFG_CORE_DYN_SHM=y
```

Comment:
I think CFG_CORE_ASLR=y CFG_TA_ASLR=y should be removed, because (1) they are the default values for QEMUv8 and (2) it should be possible to disable them if needed (they are not mandatory for virtualization, are they?)

Reply:
No they are not. I added them I was playing with different options. I will remove them.